compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -6095,16 +6095,6 @@ OMR::Z::CodeGenerator::supportsMergingOfHCRGuards()
           !self()->comp()->compileRelocatableCode();
    }
 
-bool
-OMR::Z::CodeGenerator::isConcurrentScavengeEnabled()
-   {
-#if defined(OMR_GC_CONCURRENT_SCAVENGER)
-   return TR::Compiler->target.cpu.getS390SupportsGuardedStorageFacility();
-#else
-   return false;
-#endif
-   }
```

Comment:
@dsouzai the function is already defined in the base code generator. We were only overloading it here via extensible class. See [1].

[1] https://github.com/eclipse/omr/blob/master/compiler/codegen/OMRCodeGenerator.hpp#L499-L509